ForLoop/Exercise/TrekkingMania/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace TrekkingMania
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int groupCount = int.Parse(Console.ReadLine());
+            double sumOfGroups = 0;
+            double musalaClimbers = 0.0;
+            double monblanClimbers = 0.0;
+            double kilimandjaroClimbers = 0.0;
+            double kSecondClimbers = 0.0;
+            double everestClimbers = 0.0;
```

Comment:
Are these variables referring to actual climbers? If so they should not be double - we can't have a climber and a half. If not - try to use better names